app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui";
+
+import Form from "./Form";
+
+import { CONTACT_FORM_INITIAL_FORM_VALUES } from "../constants";
+
+const NewContactPane = ({ fetchContacts, showPane, setShowPane }) => {
```

Comment:
Go through the prev review comments in this PR.

Reply:
I have rectified this. Thanks, Yedhin.